apps/ios/Shared/Views/ChatList/ChatListView.swift

```diff
@@ -80,7 +80,7 @@ struct ChatListView: View {
                 destination: chatView(),
                 isActive: Binding(
                     get: { chatModel.chatId != nil },
-                    set: { _, _ in chatModel.chatId = nil }
+                    set: { _, _ in }
```

Comment:
```suggestion
                    set: { active in
                        print(active)
                        if !active { chatModel.chatId = nil }
                    }
```

try to see what this does and when this is called. The way it was seems wrong indeed.